css/index.css

```diff
@@ -1 +1,313 @@
+h1, h2, h3, h4, h5, h6, p, blockquote, pre,
+a, abbr, acronym, address, big, cite, code,
+del, dfn, em, img, ins, kbd, q, s, samp,
+small, strike, strong, sub, sup, tt, var,
+b, u, i, center,
+dl, dt, dd, ol, ul, li,
+fieldset, form, label, legend,
+table, caption, tbody, tfoot, thead, tr, th, td,
+article, aside, canvas, details, embed, 
+figure, figcaption, footer, header, hgroup, 
+menu, nav, output, ruby, section, summary,
+time, mark, audio, video {
+	margin: 0;
+	padding: 0;
+	border: 0;
+	font-size: 100%;
+	font: inherit;
+	vertical-align: baseline;
+}
+/* HTML5 display-role reset for older browsers */
+article, aside, details, figcaption, figure, 
+footer, header, hgroup, menu, nav, section {
+	display: block;
+}
+body {
+	line-height: 1;
+}
+/* ol, ul {
+	list-style: none;
+} */
+blockquote, q {
+	quotes: none;
+}
+blockquote:before, blockquote:after,
+q:before, q:after {
+	content: '';
+	content: none;
+}
+table {
+	border-collapse: collapse;
+	border-spacing: 0;
+}
+
+/* Set every element's box-sizing to border-box */
+* {
+    box-sizing: border-box;
+}
+
+html, body {
+    height: 100%;
+    font-family: 'Titillium Web', sans-serif;
+}
+
+h1, h2, h3, h4, h5 {
+    font-family: 'Bangers', cursive;
+    letter-spacing: 1px;
+    margin-bottom: 15px;
+}
+
+/* Copy and paste your work from yesterday here and start to refactor into flexbox */
+
+
+/* Use your own code or past solution for Great Idea Web Page CSS here! */
+
+html {
+    font-size: 62.5%
+}
+.container{
+    position: relative;
+    width: 100%;
+    max-width:880px;
+    font-size: 1.6rem;
+    border: 1px solid;
+    margin: 0 auto;
+}
+nav{
+    display: flex;
+    justify-content: space-between;
+    padding-bottom: 2.3%;
+    color: gray;
+}
+nav a{
+    color: gray;
+    text-decoration: none;
+}
+nav a:link{
+    color: gray;
+}	
+nav a:visited{
+    color: gray;
+}	
+nav a:hover{
+    color: lightgray;
+}	
+.main-nav ul{
+    display: flex;
+    justify-content: space-between;
+    width: 70%;
+    padding-top: 1%;
+    list-style: none;
+}
+.main-nav li{
+    display: inline;
+    vertical-align: middle;
+}
+.banner{
+    display: flex;
+    width: 100%;
+    padding-top: 4.4%;
+    margin-bottom: 7%;
+}
+.banner h1{
+    position: relative;
+    font-size: 7.2rem;
+    text-align: center;
+    margin-top: 12%;
+}
+.banner button{
+    display: block;
+    position: relative;
+    font-size: 11pt;
+    padding: 5px 50px;
+    background-color: white;
+    border: 1px solid;
+    margin: auto;
+}
+section:first-of-type{
+    border-top: 1px solid;
+}
+section{
+    margin: 0; 
+    padding: 3.4% 0;
+}
+main img:last-of-type{
+    position: relative;
+    display: block;
+    margin-left: auto;
+    margin-right: auto;
+}
+section:last-of-type{
+    border-bottom: 1px solid;
+}
+.col-1{
+    display: flex;
+}
+.col-2{
+    display: flex;
+    justify-content: space-around;
+}
+.col-2 div{
+    width: 31.3%
+}
+.col-top{
+    width: 49%;
```

Comment:
In your tablet and mobile views, this width restricts the size of your `h1` which is why the text stops being centered as you shrink the viewport size. If you know that the container is spanning the full width of the page, just set it to `width: 100%` inside your tablet media query. That solved the issue for me :)